mart/attack/adversary.py

```diff
@@ -162,38 +181,59 @@ def forward(
         self,
         input: Union[torch.Tensor, tuple],
         target: Union[torch.Tensor, Dict[str, Any], tuple],
-        model: torch.nn.Module,
-        **kwargs
+        *,
```

Comment:
Should this be just after `self` so all arguments are keyword arguments?

Reply:
Somehow `input` and `target` are made non-keyword arguments in `Adversary`. But it should be fine to make them all required keyword arguments in`IterativeGenerator` because we explicitly make that call in `Adversary` with all keyword arguments.

Reply:
Is it worth changing `Adversary` then?

Reply:
I wouldn't change the specification of `Adversary` in this PR, because it's not backward compatible and it would require changing configurations of Adversary in sequence accordingly.